src/artrace.py

```diff
@@ -83,7 +81,9 @@ def dispdir(msframe, dispwin=None, mode=0):
             return 1
 
 
-def trace_object(slf, det, sciframe, varframe, crmask, trim=2.0, triml=None, trimr=None, sigmin=2.0, bgreg=None, maskval=-999999.9, order=0):
+def trace_object(slf, det, sciframe, varframe, crmask, trim=2.0,
+                 triml=None, trimr=None, sigmin=2.0, bgreg=None,
+                 maskval=-999999.9, order=0, doqa=True, debug=False):
```

Comment:
I'm not opposed to including a debug keyword here, but we could also use the global debug variable (which is stored in msgs._debug) and is originally set at the top of the pypit.py file.